tools/search_mawsuah.py

```diff
@@ -90,5 +90,5 @@ def pp_response(self, response):
     def run_as_list(self, query: str, num_results: int=10):
         return self.pp_response(self.run(query, num_results))
     
-    def run_as_string(self, query: str, num_results: int=10):
-        return '\n'.join(self.pp_response(self.run(query, num_results)))
+    def run_as_json(self, query: str, num_results: int=10):
```

Comment:
I think I was thinking something more structured (eg. metadata about page number and volume).